docs/proposals/org-groups.md

```diff
@@ -0,0 +1,82 @@
+	MemberGroups util.StringSet
+}
+```
+
+The Organization has a list of admins.  A default policy rule is bound to all-authenticated users that allows them inspect and modify OrgGroups of Organizations that they are an Admin for (whether via User or Group).  Having the policy rule described and bound in this way makes it easier to manage mutiple organizations without having a proliferation of roles and rolebindings.
+
+The Organization has a list of members.  Only users from this list may be assigned to OrgGroups.  This prevents an overzealous organization admin from creating OrgGroups made up of non-members.  By default, an organization admin does not have rights to change Organization membership.
+
+*****TBD*****: How are users added to the Organization???  Being able to modify the OrgGroup structure of known members seems logically distinct from being able to force users into an Organization.  I wouldn't expect the same person to do both.
+
+Organizations will have an `organizations/members` subresource that flattens the list of members by resolving the MemberGroups.
+
+Organizations will have an `organizations/groups` subresource.  A `GET` returns the list of OrgGroups.  A `POST` with a OrgGroup creates a new OrgGroup.  A `PUT` with a OrgGroup updates a new OrgGroup.  Subresources can't be named, so we do not have an individual get for `organizations/groups`.  This approach will require enhancements to `kubectl`, but prevents an Organization admin from being able to perform a `list` operation against `orggroups`.  The alternative is to have `list` on `orggroups` only show the `OrgGroups` you're allowed to see, but that would be different from all other `list`s.
+
+
+## Tying Projects to Organizations
+A Project may have at most one owner (Organization).  An Organization may own many projects.  We will indicate project ownership by having an "openshift.io/owner" label that contains a value of the form "resource/namespace/name".  Initially, the only valid resource will an Organization, so the value will look like: "organizations//MyOrg".  This allows for easy listing of projects owned by a particular Organization and allows us to extend the concept of project ownership to other resource types at a later time.
+
+
+## Using OrgGroups in RoleBindings
+RoleBindings will get a new field:
+```
+	// OrgGroups holds all the OrgGroups directly bound to the role.
+	OrgGroups util.StringSet
+```
+The `OrgGroups` field can only refer to OrgGroups owned by the Organization that owns the Project.  Since OrgGroup names are always scope qualified, any disallowed OrgGroup name will be rejected by the RESTStorage and will not be respected by our authorizer.  Our `oc policy` commands will be able to automatically scope qualifying OrgGroups.  Having a separate field makes it possible to export rolebindings from one project and create them in another project without getting rejected by admission for having invalid OrgGroup references in the Groups field.
```

Comment:
> This will be something that oc export should take care similar to what you did for sa? Also you probably mean "having invalid OrgGroup references in the OrgGroups field."

RoleBindings have moved on to `Subjects` that are `ObjectReferences`, so that would now be possible